science/imager/Portfile

```diff
@@ -26,11 +26,11 @@ set site_base       https://nuage.oasu.u-bordeaux.fr/index.php/s/PoxdxMTTYXqNwKF
 master_sites        ${site_base}/download?path=/&files=
 distname            patched-macport-${name}-${my_version}
 
-checksums           rmd160  bc5774738e891068e4b074da928529c57c14844b \
-                    sha256  b63123fe82093d3c76778014f2c2310a17e8eefa78b5d8b9440611e3a9e9aadf \
-                    size    32483166
+checksums           rmd160  9f09c675e43492261f196c7de11a8efb252ee736 \
+                    sha256  4187c417c841ee010910948f4751d92ce0e2a187cbfb856760a2eb377dae2b99 \
+                    size    32496624
 
-set py_ver          3.12
+set py_ver          3.11
```

Comment:
Was there some problem with python 3.12? If so, what was it?

Generally we want to use python 3.12 now, but of course if it doesn't work then using 3.11 is fine.

Reply:
hi,
yes I have to downgrade to version 3.11 for the moment, segfault with 3.12
Thanks

Reply:
do you mean that python 3.12 always segfaults on your system? Or this specific software is incompatible with python 3.12 on all systems, for some reason?

Reply:
The researcher who develops Imager will correct the small problems linked to 3.12 (bugs on a functionality) in a future version. But his program works correctly with 3.11. Can you validate the merge request so that users can benefit from the latest imager updates ? thanks

Reply:
OK, so you are confirming that this software will not work with python 3.11 on any system at present. Thanks, that's what we were looking to understand.